esp-hal/src/analog/adc/esp32.rs

```diff
@@ -194,9 +194,7 @@ impl RegisterAccess for ADC2 {
 /// Analog-to-Digital Converter peripheral driver.
 pub struct ADC<'d, ADC> {
     _adc: PeripheralRef<'d, ADC>,
-    #[allow(dead_code)] // FIXME
```

Comment:
This was already fixed by removing the dep on the eh2 deps, thanks @jessebraham :sunglasses: